dm/openapi/spec/dm.yaml

```diff
@@ -1376,7 +1376,7 @@ components:
           type: string
           example: "123456"
           description: "source password"
-          nullable: false
+          nullable: true
```

Comment:
also need to remove reqired filed `password` for source, 
it seems that openapi will return 400 when input  json don't have this filed, maybe add a it-test for this

Reply:
yes. i forget. i will fix and add a it-test.